### ISSUE TITLE
No explaination of where the setting is for Teams

### ISSUE THREAD

**Issue by brianleesmith** [MicrosoftDocs/microsoft-365-docs#12356]
There doesn't seem to be an explanation of where to set the ZAP policy for Teams, only email.  Unless configuring the anti-spam and anti-malware policies affect both.  If that is the case, please update this article indicating that AND also the help in admin pages ("i" next to settings).

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4fb415a2-fc7c-5b9a-96de-f01be631bf07
* Version Independent ID: c5d4ab93-4f59-a60f-5d42-1a6a52a37f4c
* Content: [Zero-hour auto purge in Microsoft Defender for Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/zero-hour-auto-purge?view=o365-worldwide#zero-hour-auto-purge-zap-in-microsoft-teams)
* Content Source: [microsoft-365/security/office-365-security/zero-hour-auto-purge.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/zero-hour-auto-purge.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@brianleesmith, I don't think there's any setting associated with ZAP for Teams (Defender for Office 365 for Teams is turned on = ZAP for Teams is turned on). Do we need to say that in the article?

**Comment by brianleesmith** [comment]
I would probably say yes to it needing to be stated.  Of course, it would be great if it was explicitly stated in the admin screens as well…but that’s not you. :-)Thanks for the amazingly quick reply.On Jun 30, 2023, at 1:27 PM, Chris Davis ***@***.***> wrote:﻿
@brianleesmith, I don't think there's any setting associated with ZAP for Teams (Defender for Office 365 for Teams is turned on = ZAP for Teams is turned on). Do we need to say that in the article?

**Comment by chrisda** [comment]
@brianleesmith , OK, I added the following statement to the NOTE at the beginning of the [Zero-hour auto purge (ZAP) in Microsoft Teams](https://learn.microsoft.com/microsoft-365/security/office-365-security/zero-hour-auto-purge?view=o365-worldwide#zero-hour-auto-purge-zap-in-microsoft-teams):

> When you [join the Preview](https://learn.microsoft.com/microsoft-365/security/office-365-security/mdo-support-teams-about#enable-microsoft-defender-for-teams), ZAP for Microsoft Teams is turned on.

If you would like to modify that statement after it publishes, feel free to do so. Instructions are [here](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/README.md).

Thanks for the feedback.

Closing
